tests/02_00_06-2wp-new-minimum.js

```diff
@@ -1,117 +1,130 @@
 
-const MIN_PEGOUT_VALUE_IN_SATOSHIS = 250000;
-const BTC_TX_FEE_IN_SATOSHIS = 100000;
+describe('2wp after iris300, using new minimum values', () => {
+    let rskTxHelpers;
+    let btcTxHelper;
+    let rskTxHelper;
+    let MINIMUM_PEGIN_VALUE_IN_SATOSHI;
+    let MINIMUM_PEGIN_VALUE_IN_BTC;
+    let bridge;
 
-let btcClient;
-let rskClient;
-let pegUtils;
-let minimumPeginValueInSatoshis;
+    const fulfillRequirementsToRunAsSingleTestFile = async () => {
+        await rskUtils.activateFork(Runners.common.forks.fingerroot500);
+    };
 
-describe('2wp after iris300, using new minimum values', () => {
     before(async () => {
-        btcClient = btcClientProvider.getBtcClient();
-        rskClient = rsk.getClient(Runners.hosts.federate.host);
-        pegUtils = _2wpUtilsLegacy.with(btcClient, rskClient);
+        rskTxHelpers = getRskTransactionHelpers();
+        rskTxHelper = rskTxHelpers[0];
+        btcTxHelper = getBtcClient();
+  
+        if(process.env.RUNNING_SINGLE_TEST_FILE) {
+            await fulfillRequirementsToRunAsSingleTestFile();
+        };
 
         // Get the current peg-in minimum
-        minimumPeginValueInSatoshis = await rskClient.rsk.bridge.methods.getMinimumLockTxValue().call();
+        const latestActiveForkName = await getLatestActiveForkName();
+        bridge = getBridge(rskTxHelper.getClient(), latestActiveForkName);
+        MINIMUM_PEGIN_VALUE_IN_SATOSHI = await bridge.methods.getMinimumLockTxValue().call();
+        MINIMUM_PEGIN_VALUE_IN_BTC = btcEthUnitConverter.satoshisToBtc(MINIMUM_PEGIN_VALUE_IN_SATOSHI);
     });
     
     it('should peg-in when sending minimum value', async () => {
-        const AMOUNT_TO_LOCK_IN_BTC = btcClient.nodeClient.satoshisToBtc(minimumPeginValueInSatoshis);
-
         // Create legacy type address to use as sender
-        let senderAddressInformation = await btcClient.generateBtcAddress('legacy');
-        let initialSenderBalance = await btcClient.getAddressBalance(senderAddressInformation.address);
+        const senderAddressInfo = await btcTxHelper.generateBtcAddress('legacy');
+        const senderAddressBalanceInitial = Number(await btcTxHelper.getAddressBalance(senderAddressInfo.address));
+        expect(senderAddressBalanceInitial).to.equal(0);
 
         // Get the RSK address where the funds should be locked to
-        let rskDerivedAddress = keyUtils.getRskAddress(senderAddressInformation.privateKey);
-        let initialDerivedAddressBalance = await rskClient.eth.getBalance(rskDerivedAddress);
-
-        await btcClient.fundAddress(senderAddressInformation.address, AMOUNT_TO_LOCK_IN_BTC + btcClient.getFee());
+        const recipientRskAddressInfo = getDerivedRSKAddressInformation(senderAddressInfo.privateKey, btcTxHelper.btcConfig.network);
+        const recipientRskAddressBalanceInitial = Number(await rskTxHelper.getBalance(recipientRskAddressInfo.address));
+        expect(recipientRskAddressBalanceInitial).to.equal(0);
```

Comment:
Assert The Fed Balance Before And After Pegin

Reply:
you mean federation address balance? 
hmm i'm testing peg-in with minimal value. so if i receive what i expect in the `recipientRskAddress` that i think should be enough. 
i believe there are specific tests for validating balance on federation address.